src/BinDeps.jl

```diff
@@ -47,7 +47,7 @@ module BinDeps
     function download_cmd(url::AbstractString, filename::AbstractString)
         global downloadcmd
         if downloadcmd === nothing
-            for download_engine in @windows? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
+            for download_engine in (is_windows() ? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch))
```

Comment:
Without the parentheses around the whole ternary this errors with 
```
ERROR: LoadError: syntax: space before "(" not allowed in "? ("
 in include_from_node1(::String) at .\loading.jl:426
 in macro expansion; at .\none:2 [inlined]
 in anonymous at .\<missing>:?
 in eval(::Module, ::Any) at .\boot.jl:225
 in process_options(::Base.JLOptions) at .\client.jl:243
 in _start() at .\client.jl:322
while loading C:\Users\...\.julia\v0.5\BinDeps\src\BinDeps.jl, in expression starting on line 2
```
but works fine if there's no space... Not quite sure what to make of it.